pkg/tunneler/tunnel.go

```diff
@@ -96,6 +97,22 @@ func (rp *tunnelPool) deleteDialer(cluster, syncer string) {
 	delete(rp.pool, id)
 }
 
+// SyncerTunnelProxyPath builds the path for the proxy handler as expected by the Dialer
+func SyncerTunnelProxyPath(syncTargetWorkspaceName logicalcluster.Name, syncTargetName, downstreamNamespaceName, podName, subresource, arguments string) (url.URL, error) {
```

Comment:
Is the function here used only once ?
Ten shouldn't it be private inside the use-site package ?